pubsub/setup.py

```diff
@@ -52,6 +52,7 @@
 
 REQUIREMENTS = [
     'google-api-core[grpc] >= 0.1.2, < 0.2.0dev',
+    'grpcio >= 1.8.2',
```

Comment:
@lukesneeringer How bad an idea is it to put this requirement here? Should I just cut a new release of `google-api-core` rather than depending on "future me" to remove this?

Reply:
If the pub/sub library explicitly imports `grpcio`, I would actually prefer the requirement be explicitly stated.